lib/libmbedtls/mbedtls/library/ssl_tls.c

```diff
@@ -187,6 +187,10 @@ int mbedtls_ssl_session_copy( mbedtls_ssl_session *dst,
     mbedtls_ssl_session_free( dst );
     memcpy( dst, src, sizeof( mbedtls_ssl_session ) );
 
+#if defined(MBEDTLS_SSL_SESSION_TICKETS) && defined(MBEDTLS_SSL_CLI_C)
```

Comment:
all of this change come from CVE patch? If yes, please remove your signed-off-by.

Reply:
No, this CVE patch can't be used directly, we patch it manually. So two names are remained.

Reply:
Then can we just change the code manually? I mean just append your signed-off-by and try to make the title shorter.

Reply:
@GangSecurity , how to handle this case previously? Is there any precedents？